lib/gateway/pipelines.js

```diff
@@ -223,14 +228,16 @@ function validatePipelinePolicies (policies, config) {
   let policiesConfig = config.gatewayConfig.policies || [];
 
   policies.forEach(policyObj => {
-    let policyNames = Object.keys(policyObj);
+    if (policyObj && typeof policyObj === 'object') {
```

Comment:
how this can be ? 
if there is no policies just do not call  policies.forEach, why this check in foreach ?

Reply:
if the idea is to protect from some invalid policy configuration please add some validation logic